.github/workflows/ci-new.yml

```diff
@@ -160,13 +161,23 @@ jobs:
           service_account: '${{ env.WIF_SERVICE_ACCOUNT }}'
       - name: 'Setup gcloud'
         uses: 'google-github-actions/setup-gcloud@e30db14379863a8c79331b04a9969f4c1e225e0b' # ratchet:google-github-actions/setup-gcloud@v1
+      - name: 'create_kms_key'
```

Comment:
Do we need to cleanup the keys?

Reply:
I am thinking we will have a dedicated workflow to auto cleanup the cloud run revisions, we should cleanup the keys in the same place as they are both cleanup work.

Reply:
Did a quick search, a key cannot be deleted, and we are charged for the number of active key versions, we should disable the key version in each run for cleanup purpose. It seems easier to do it here in this workflow. Also have you check is there a limit on how many keys we can have in a key ring, or if any issues with creating a lot of keys in a single key ring?

Reply:
Oh I will take a look on this and update the PR if necessary

Reply:
Updated the code, now we destroy keys in the CI workflow.
See: https://screenshot.googleplex.com/3AaQm3vUjHXUqLK